kapacitor/client.go

```diff
@@ -57,23 +57,117 @@ type Task struct {
 	TICKScript chronograf.TICKScript // TICKScript is the running script
 }
 
+// NewTask creates a task from a kapacitor client task
+func NewTask(task *client.Task) *Task {
+	dbrps := make([]chronograf.DBRP, len(task.DBRPs))
+	for i := range task.DBRPs {
+		dbrps[i].DB = task.DBRPs[i].Database
+		dbrps[i].RP = task.DBRPs[i].RetentionPolicy
+	}
+
+	script := chronograf.TICKScript(task.TICKscript)
+	rule, err := Reverse(script)
+	if err != nil {
```

Comment:
Should this be surfaced somehow? IIRC this means that it's a TICKScript that we don't support with the frontend machinery—Is an `AlertRule` with a `nil` query enough to communicate that?

Reply:
Hey @timraymond , the nil query is how we have done it previously, but, you are correct.  Long term that isn't the right solution.  Long term, though, there will not be a queryConfig at all.  This will help us transition away from queryConfig I think.